src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -107,8 +107,28 @@ public static ColumnSelectingEstimator DropColumns(this TransformsCatalog catalo
         /// </example>
         public static ColumnSelectingEstimator SelectColumns(this TransformsCatalog catalog,
             string[] keepColumns,
-            bool keepHidden = ColumnSelectingTransformer.Defaults.KeepHidden)
+            bool keepHidden)
             => new ColumnSelectingEstimator(CatalogUtils.GetEnvironment(catalog),
                 keepColumns, null, keepHidden, ColumnSelectingTransformer.Defaults.IgnoreMissing);
+
+        /// <summary>
+        /// ColumnSelectingEstimator is used to select a list of columns that user wants to keep from a given input.
```

Comment:
Could you use the same comment across the two extensions?
Also I would remove the reference to user, maybe something like: 

```suggestion
        /// Select a list of columns to keep in a given <see cref="IDataView"/>.
```

Reply:
Yes, good catch!

---
In reply to: [254465617](https://github.com/dotnet/machinelearning/pull/2439#discussion_r254465617) [](ancestors = 254465617)